packages/plugin-ext/src/plugin/types-impl.ts

```diff
@@ -390,7 +411,8 @@ export class Range {
         return new Range(start, end);
     }
 
-    static isRange(thing: {}): thing is theia.Range {
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
+    static isRange(thing: any): thing is theia.Range {
```

Comment:
Why is this change necessary?

Reply:
The same reason: to make this class to be defined exactly as it's done in VS Code. Probably it's not necessary change.

Reply:
I can rollback the changes in types-impl.ts as they aren't necessary and do not bring any fixes/improvements regarding this issue. Rolledback in fixup